GameVersion.h

```diff
@@ -50,6 +50,10 @@ class GameVersion: commonItems::convenientParser
 
 	friend std::ostream& operator<<(std::ostream&, const GameVersion& version);
 
+	[[nodiscard]] static std::optional<GameVersion> extractLauncherVersion(const std::string& filePath); // modern PDX games, scrapes launcher-settings.json.
+	[[nodiscard]] static std::optional<GameVersion> extractReadMeVersion(const std::string& filePath); // extracts version from Vic2 ReadMe.txt/Readme.txt.
+	[[nodiscard]] static std::optional<GameVersion> extractChangeLogVersion(const std::string& filePath); // extracts version from CK2 ChangeLog.txt.
+
```

Comment:
Maybe rename these to `extractVersionFromX`? `extractLauncherVersion` sounds like you're checking the launcher's version.

Reply:
yup yup